dlt/destinations/impl/postgres/postgres.py

```diff
@@ -112,6 +112,10 @@ def __init__(self, table_name: str, file_path: str, sql_client: Psycopg2SqlClien
         with FileStorage.open_zipsafe_ro(file_path, "rb") as f:
             # all headers in first line
             headers = f.readline().decode("utf-8").strip()
+            # quote headers if not quoted - all special keyword like "binary" must be quoted
+            headers = ",".join(
+                h if h.startswith('"') else '"' + h + '"' for h in headers.split(",")
```

Comment:
I think we need to use `f`-strings for readbility

Reply:
OK I agree, will change it